Answers.md

```diff
@@ -1 +1,19 @@
+1. Describe the biggest difference between `.forEach` & `.map`.
+`.forEach` executes a provided function once for each array element.
 
+`.map` creates a new array with the results of calling a provided function on every element in the calling array.
+
+2. What is the difference between a function and a method?
+A function is a piece of code that is called by name but a method is a piece of code that is called by a name that is associated with an object.
+
+3. What is closure?
+it's where inner functions have access to the outer enclosing function's variables.
+
+4. Describe the four rules of the 'this' keyword.
+1) Window/Global Object Binding
+2) Implicit Binding
```

Comment:
James, I think that the intent of this question was to give you an opportunity to _describe_ the four rules, and not just name them. If you don't feel like you can explain each of the four rules, please take some time to review these!